aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -840,6 +840,22 @@ connect:
   tags:
     product_categories: {}
   version: connect-2017-08-08
+controltower:
+  api_ref: controltower/latest/APIReference/Welcome.html
+  blurb: ''
```

Comment:
Ideally you include a blurb for WRITEME to use.

Reply:
Fixed.